web/test/pages/index.test.tsx

```diff
@@ -35,33 +35,33 @@ describe("HomePage", () => {
         onboardingFormProgress: "COMPLETED",
         profileData: generateProfileData({ businessPersona: "STARTING" }),
       });
-      render(withAuth(<Home />, { user: generateUser({}) }));
+      render(withAuth(<Home isWelcomePage={false} />, { user: generateUser({}) }));
```

Comment:
If we make `isWelcomePage` an optional param, you can drop it from these render statements.